apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/AssessmentDetailsDialog.tsx

```diff
@@ -607,16 +612,29 @@ const AssessmentDetailsDialog = ({
                     })}
                   </div>
                 </div>
-                <Checklist
-                  idPrefix="assessedSkills"
-                  id="assessedSkills"
-                  name="assessedSkills"
-                  legend={intl.formatMessage(labels.assessedSkills)}
-                  items={assessedSkillsItems}
-                  rules={{
-                    required: intl.formatMessage(errorMessages.required),
-                  }}
-                />
+                {selectedTypeOfAssessment ===
+                  AssessmentStepType.ScreeningQuestionsAtApplication && (
+                  <Checklist
+                    idPrefix="assessedSkillsScreeningQuestions"
+                    id="assessedSkillsScreeningQuestions"
+                    name="assessedSkillsScreeningQuestions"
+                    legend={intl.formatMessage(labels.assessedSkills)}
+                    items={assessedSkillsItems}
+                  />
+                )}
+                {selectedTypeOfAssessment !==
+                  AssessmentStepType.ScreeningQuestionsAtApplication && (
+                  <Checklist
```

Comment:
One conditional input wasn't working, as far as I could tell the rules object somehow appeared to be persisted across both iterations.

Reply:
If it's acting weird, though, I'm fine to leave this structure.